exams/utils.py

```diff
@@ -66,9 +66,24 @@ def is_eligible_for_exam(mmtrack, course_run):
         course_run (courses.models.CourseRun): A CourseRun object.
 
     Returns:
-        bool: whether use is eligible or not
+        bool: whether user is eligible or not
     """
-    return course_run.has_exam and mmtrack.has_paid(course_run.edx_course_key)
+    return course_run.has_exam and has_paid(mmtrack, course_run.edx_course_key)
+
+
+def has_paid(mmtrack, edx_course_key):
+    """
+    Returns True if user has paid for the course or payment check is suppressed using feature flag
+    FEATURE_SUPPRESS_PAYMENT_FOR_EXAM.
+    Args:
+        mmtrack (dashboard.utils.MMTrack): a instance of all user information about a program.
+        edx_course_key (str): An edX course key (CourseRun.edx_course_key)
+
+    bool: whether user has paid or not.
+    """
+    # if FEATURE_SUPPRESS_PAYMENT_FOR_EXAM=True then bypass payment check
+    suppress_payment_for_exam = settings.FEATURES.get("SUPPRESS_PAYMENT_FOR_EXAM", True)
+    return mmtrack.has_paid(edx_course_key) or suppress_payment_for_exam
```

Comment:
Could you reverse the order of these so we don't check `mmtrack.has_paid()` if `suppress_payment_for_exam` is already True.